regexes/oss.yml

```diff
@@ -5,6 +5,13 @@
 # @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
 ###############
 
+##########
+# PureOS (https://pureos.net/)
+##########
+- regex: 'PureBrowser'
+  name: 'PureOS'
+  version: ''
```

Comment:
we cannot rely on this definition
1 Android browser pure https://purebrowser.me/  (this is a different browser)
2 PureBrowser is no longer supported and has been replaced in PureOS  - Firefox-ESR

Reply:
installed AndroiStudio and PureBrowser in the emulator.
app: `https://play.google.com/store/apps/details?id=pure.lite.browser`

useragent: `Mozilla/5.0 (Linux; Android 11; sdk_gphone_x86_64 Build/RSR1.201211.001) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/83.0.4103.106 Mobile Safari/537.36`